Yoast/wordpress-seo
# Fix alignment issues go-premium notice in social tab

## Summary

This PR can be summarized in the following changelog entry:

* Fixed alignment of Go Premium notice.

## Relevant technical choices:

* Made the Go Premium notice margins 0.

## Test instructions

This PR can be tested by following these steps:

* Checkout a post in the classic editor and in Gutenberg. Notice that in Gutenberg the alignment is different due to 15px left and right margins.
* Checkout the branch and notice the difference is gone: in Gutenberg it should look like in the classic metabox.

## Quality assurance


Fixes https://github.com/Yoast/gutenberg/issues/39
